### ISSUE TITLE
Error installing alcotest

### ISSUE THREAD

**Issue by jdjakub** [ocaml/opam-repository#6943]
```
jdj27@jdj27-VirtualBox:~/angstrom$ opam install alcotest
The following actions will be performed:
 - install   topkg.0.7.6                           [required by alcotest]
 - install   fmt.0.8.0                             [required by alcotest]
 - install   alcotest.0.6.0
=== 3 to install ===
Do you want to continue ? [Y/n] y

=-=- Synchronizing package archives -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[default] Downloading https://opam.ocaml.org/archives/alcotest.0.6.0+opam.tar.gz
[default] Downloading https://opam.ocaml.org/archives/fmt.0.8.0+opam.tar.gz
[default] Downloading https://opam.ocaml.org/archives/topkg.0.7.6+opam.tar.gz

=-=- Installing packages =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Building topkg.0.7.6:
  ocaml pkg/pkg.ml build --pkg-name topkg --pinned false
Installing topkg.0.7.6.
Building fmt.0.8.0:
  ocaml pkg/pkg.ml build --pinned false --with-base-unix true --with-cmdliner true
Installing fmt.0.8.0.
[ERROR] 'pinned' has type bool, but a env element of type string was expected.
Building alcotest.0.6.0:
  ocaml pkg/pkg.ml build --pinned
[ERROR] The compilation of alcotest.0.6.0 failed.
Removing alcotest.0.6.0.
  Nothing to do.


#=== ERROR while installing alcotest.0.6.0 ====================================#
# opam-version 1.2.0
# os           linux
# command      ocaml pkg/pkg.ml build --pinned
# path         /home/jdj27/.opam/4.03.0/build/alcotest.0.6.0
# compiler     4.03.0
# exit-code    1
# env-file     /home/jdj27/.opam/4.03.0/build/alcotest.0.6.0/alcotest-4828-ad8886.env
# stdout-file  /home/jdj27/.opam/4.03.0/build/alcotest.0.6.0/alcotest-4828-ad8886.out
# stderr-file  /home/jdj27/.opam/4.03.0/build/alcotest.0.6.0/alcotest-4828-ad8886.err
### stderr ###
# pkg.ml: [ERROR] key --pinned: No value specified.


=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
These actions have been completed successfully
 - install   fmt.0.8.0
 - install   topkg.0.7.6
The following failed
 - install   alcotest.0.6.0

The former state can be restored with opam switch import "/home/jdj27/.opam/4.03.0/backup/state-20160611103215.export"
```

**Comment by dbuenzli** [comment]
This the same problem as https://github.com/ocaml/opam-repository/pull/6888#issuecomment-229821305. Apparently it went through CI without notice here. If you need a quick fix, either upgrade to OPAM `1.2.2` or do a: 

```
opam pin add alcotest --dev-repo
opam pin edit alcotest # substitute pinned by "%{pinned}%"
```

**Comment by samoht** [comment]
This is now fixed.
